src/views/v2/list/nav/next.php

```diff
@@ -11,17 +11,30 @@
  *
  * @var string $link The URL to the next page, if any, or an empty string.
  *
- * @version 4.9.3
+ * @version TBD
  *
  */
 ?>
 <li class="tribe-events-c-nav__list-item tribe-events-c-nav__list-item--next">
 	<a
 		href="<?php echo esc_url( $link ); ?>"
 		rel="next"
-		class="tribe-events-c-nav__next tribe-common-b2"
+		class="tribe-events-c-nav__next tribe-common-b2 tribe-common-b1--min-medium"
 		data-js="tribe-events-view-link"
 	>
-		<?php echo esc_html( sprintf( __( 'Next %s', 'the-events-calendar' ), tribe_get_event_label_plural() ) ); ?>
+		<?php
+			echo sprintf(
+				wp_kses(
+					/* translators: %s: Event (plural or singular). */
+					__( 'Next<span class="tribe-events-c-nav__next-label-plural"> %s</span>', 'the-events-calendar' ),
+					array(
+						'span' => array(
+							'class' => array(),
+						),
+					)
+				),
+				tribe_get_event_label_plural()
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 5234d4b1e5ed3ea958207779b1892fe3</sub>